.circleci/config.yml

```diff
@@ -0,0 +1,74 @@
+# Use the latest 2.1 version of CircleCI pipeline process engine. See: https://circleci.com/docs/2.0/configuration-reference
+version: 2.1
+
+workflows:
+  build:
+    jobs:
+      - lint:
+          matrix:
+            parameters:
+              dir:
+                - promql
+                - remote_write
+      - test:
+          matrix:
+            parameters:
+              dir:
+                - promql
+                - remote_write
+
+jobs:
+  test:
+    parameters:
+      dir:
+        type: string
+    working_directory: ~/project/<< params.dir >>
```

Comment:
parameters.dir

Reply:
I don't actually think this will work, as IIUC this is also the directory it'll try and clone into...

Reply:
`checkout` triggers circleci to checkout the repo into ~/project. This is the default working_directory.

Reply:
https://circleci.com/docs/2.0/configuration-reference/#checkout

> A special step used to check out source code to the configured path (defaults to the working_directory).

Reply:
Ohhhhhhhhhhhhh, nice. I missed that bit.